frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts

```diff
@@ -280,6 +281,15 @@ export const tranformKnNodeData = (
   let edgesData = [];
   let groupsData = [];
   const dataToShowOnNodes = {};
+  const serviceBindingRequests = _.get(resources, 'serviceBindingRequests.data');
+  const allResources = _.cloneDeep(
+    _.concat(
+      resources.deploymentConfigs && resources.deploymentConfigs.data,
+      resources.deployments && resources.deployments.data,
+      resources.statefulSets && resources.statefulSets.data,
+      resources.daemonSets && resources.daemonSets.data,
+    ),
```

Comment:
Looks like this is a copy of what's done in the dev-console topology utils. Unsure why it's a deep clone.
We should revisit this as well as find a way to perhaps create edges by constructing the source / target IDs instead of having to resolve the resource to get its uid.